app/assets/javascripts/jcrop_data_objects.js

```diff
@@ -48,14 +48,41 @@ $(function(){
     });
   }
 
-  function updatePreviewForm(c)
+  function updatePreviewForm(coords)
   {
-    $crop_form.children('[name="x"]').val(c.x);
-    $crop_form.children('[name="y"]').val(c.y);
-    $crop_form.children('[name="w"]').val(c.w);
-    $crop_form.children('[name="h"]').val(c.h);
+    var w = $target_image[0].naturalWidth;
+    var h = $target_image[0].naturalHeight;
+    if (typeof w == "undefined") {
+      // IE 6/7/8 doesn't define naturalWidth etc, so load up a hidden copy to get the orig widths
+	  var newImg = new Image();
+	  newImg.onload = function() {fillForm(this.width, this.height, coords);};
+	  newImg.src = $target_image[0].src;
+    } else {
+      fillForm(w, h, coords);
+    }
   }
-
+  
+  function fillForm(w,h,c)
+  {
+    //EoL specific: export the crop as percentages, since large images may be shrunk
+    // Offsets are from the 580 x 360 image. However, if they are wider than 
+    // 540px, the EoL CSS scales the image proportionally to fit into a max width of 540.
+    // The offsets and width need to be scaled to match the image dimensions
+    var scale_factor = 1;
+    if((w / h) < ( 540 / 360 ))
+    {
+      //smaller width, so scaling only happens if height exceeds max
+      if(h > 360) scale_factor = h / 360;
```

Comment:
Identifier 'scale_factor' is not in camel case.<br>Expected '{' and instead saw 'scale_factor'.